src/recipes/mango-yakult-juice.json

```
@@ -0,0 +1,40 @@
+{
+    "name": "Mango Juice with Yakult",
+    "description": "Mango Juice with mix of Yakult.",
+    "github": "muthi08",
+    "ingredients": [
+        {
+            "quantity": "1",
+            "measure": "",
+            "ingredient": "Mango"
+        },
+        {
+            "quantity": "2",
+            "measure": "bottle",
+            "ingredient": "Yakult"
+        },
+        {
+            "quantity": "1",
+            "measure": "tbs",
+            "ingredient": "Chiaseed"
+        },
+        {
+            "quantity": "1",
+            "measure": "sachet",
+            "ingredient": "Tropicana"
+        }
+    ],
+    "directions": [
+        "peel mango's skin troughly.",
+        "Cut the mango into cubes and add them to a blender and pour a bottle of Yakult, a sachet of Tropicana and 100 ml of water.",
+        "Blend until smooth. Serve immediately.",
+        "Enjoy!"
+    ],
+    "image": "mano-yakult-juice.jpg",
```

```suggestion
    "image": "mango-yakult-juice.jpg",
```

Also, is it possible to use a horizontal image too?